### PR TITLE
update users

### PR DESCRIPTION
**Pull request by VuongLong**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <nav>
       <ul class="nav navbar-nav navbar-right">
         <% if logged_in? %>
-          <li><%= link_to t("static_pages.users"), "#" %></li>
+          <li><%= link_to t("static_pages.users"), users_path %></li>
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Use link_to

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -12,15 +12,13 @@
...
         <% if logged_in? %>
-          <li><%= link_to t("static_pages.users"), "#" %></li>
+          <li><%= link_to t("static_pages.users"), users_path %></li>
           <li class="dropdown">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-              <%= t "static_pages.account" %><b class="carret"></b>
-            </a>
-            
+            <%= link_to t("static_pages.account"), "#", class: "dropdown-toggle", 
+                                                        data: {toggle:"dropdown"}%>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Insert space after }

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -18,10 +24,38 @@ def create
       render "new"
     end
   end
+  
+  def edit
+  end
+
+  def update
+    if @user.update_attributes user_params 
+      redirect_to @user
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: flash message?

File: app/helpers/sessions_helper.rb [hunk 4]
```diff
@@ -36,8 +36,17 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
-
-  def current_user? user
+  
+  def current_user? user
```
Comment (dieunb) [inline app/helpers/sessions_helper.rb]: remove trailing spaces

File: app/helpers/sessions_helper.rb [hunk 5]
```diff
@@ -36,8 +36,17 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
-
-  def current_user? user
+
```
Comment (dieunb) [inline app/helpers/sessions_helper.rb]: remove trailing spaces

**Pull request merged**